index.html

```diff
@@ -270,87 +329,289 @@
+rights in a relevant jurisdiction. (For instance, under [[?GDPR]], the [=person=] may be
+conveying objections to [=processing=] based on legitimate interest or the withdrawal of
+[=consent=] to specific [=purposes=].) It should be noted that, since a [=global opt-out=]
+signal is reaffirmed automatically with every interaction, it will take precedence
+in terms of specificity over any manner of blanket [=consent=] that a site may obtain,
+unless that [=consent=] is directly attached to an interaction (eg. terms specified on a
+form upon submission).
+
+### Privacy Labour {#privacy-labour}
+
+<dfn data-lt="privacy labor">Privacy labour</dfn> is the practice of having a [=person=] carry out
+the work of ensuring [=data processing=] of which they are the subject or recipient is
+[=appropriate=], instead of having the [=parties=] be responsible for that work.
+Data systems that are based on asking [=people=] for their [=consent=] tend to increase
+[=privacy labour=].
+
+More generally, implementations of [=privacy=] are often dominated by self-governing approaches that
+offload labour to [=people=]. This is notably true of the regimes descended from the
+<dfn data-lt="FIPs">Fair Information Practices</dfn> ([=FIPs=]), a loose set of principles initially
+elaborated in the 1970s in support of individual [=autonomy=] in the face of growing concerns with databases. The
+[=FIPs=] generally assume that there is sufficiently little [=data processing=] taking place that any
+[=person=] will be able to carry out sufficient diligence to enable [=autonomy=] in their
+decision-making. Since they entirely offload the [=privacy labour=]
+to people and assume perfect, unlimited [=autonomy=], the [=FIPs=] do not forbid specific
+types of [=data processing=] but only place them under different procedural requirements.
+Such an approach is [=appropriate=] for [=parties=] that are processing data in the 1970s.
+
+One notable issue with procedural, self-governing approaches to privacy is that they tend to have the same
+requirements in situations where people find themselves in a significant asymmetry of
+power with a [=party=] — for instance a [=person=] using an essential service provided by a
+monopolistic platform — and those where people and [=parties=] are very much on equal
+footing, or even where the [=person=] may have greater power, as is the case with small
+businesses operating in a competitive environment. It further does not consider cases in
+which one [=party=] may coerce other [=parties=] into facilitating its [=inappropriate=]
+practices, as is often the case with dominant players in advertising or
+in content aggregation ([[?Consent-Lackeys]], [[?CAT]]).
+
+Reference to the [=FIPs=] survives to this day. They are often referenced as <em>transparency
+and choice</em>, which, in today's digital environment, is often a strong indication that
+[=inappropriate=] [=processing=] is being described.
```

Comment:
Nit: ​Acknowledging the existence of bad actors and bad business practices is fine, but this statement goes beyond that. It suggests a cynical assumption of "guilty until proven innocent." I don't think we are there yet. I think we need to ​maintain a presumption of good intentions and fair-minded actors for entities that are participating as legitimate businesses.

Reply:
The intent isn't "guilty until proven innocent" but rather that a focus on transparency and control is a hallmark of inappropriate processing (which is often carried out by otherwise legitimate businesses). I've removed "strong" if that works?